itests/kit/node_unmanaged.go

```diff
@@ -24,3 +60,730 @@ type TestUnmanagedMiner struct {
 		PrivKey libp2pcrypto.PrivKey
 	}
 }
+
+type WindowPostResp struct {
+	SectorNumber abi.SectorNumber
+	Error        error
+}
+
+func NewTestUnmanagedMiner(t *testing.T, full *TestFullNode, actorAddr address.Address, opts ...NodeOpt) *TestUnmanagedMiner {
+	require.NotNil(t, full, "full node required when instantiating miner")
+
+	options := DefaultNodeOpts
+	for _, o := range opts {
+		err := o(&options)
+		require.NoError(t, err)
+	}
+
+	privkey, _, err := libp2pcrypto.GenerateEd25519Key(rand.Reader)
+	require.NoError(t, err)
+
+	require.NotNil(t, options.ownerKey, "owner key is required for initializing a miner")
+
+	peerId, err := peer.IDFromPrivateKey(privkey)
+	require.NoError(t, err)
+	tmpDir := t.TempDir()
+
+	cacheDir := filepath.Join(tmpDir, fmt.Sprintf("cache-%s", actorAddr))
+	unsealedSectorDir := filepath.Join(tmpDir, fmt.Sprintf("unsealed-%s", actorAddr))
+	sealedSectorDir := filepath.Join(tmpDir, fmt.Sprintf("sealed-%s", actorAddr))
+
+	_ = os.Mkdir(cacheDir, 0755)
+	_ = os.Mkdir(unsealedSectorDir, 0755)
+	_ = os.Mkdir(sealedSectorDir, 0755)
+
+	tm := TestUnmanagedMiner{
+		t:                 t,
+		options:           options,
+		cacheDir:          cacheDir,
+		unsealedSectorDir: unsealedSectorDir,
+		sealedSectorDir:   sealedSectorDir,
+
+		unsealedSectorPaths: make(map[abi.SectorNumber]string),
+		cacheDirPaths:       make(map[abi.SectorNumber]string),
+		sealedSectorPaths:   make(map[abi.SectorNumber]string),
+		sealedCids:          make(map[abi.SectorNumber]cid.Cid),
+		unsealedCids:        make(map[abi.SectorNumber]cid.Cid),
+		sealTickets:         make(map[abi.SectorNumber]abi.SealRandomness),
+
+		ActorAddr:        actorAddr,
+		OwnerKey:         options.ownerKey,
+		FullNode:         full,
+		currentSectorNum: 101,
+		proofType:        make(map[abi.SectorNumber]abi.RegisteredSealProof),
+	}
+	tm.Libp2p.PeerID = peerId
+	tm.Libp2p.PrivKey = privkey
+
+	return &tm
+}
+
+func (tm *TestUnmanagedMiner) AssertNoPower(ctx context.Context) {
+	p := tm.CurrentPower(ctx)
+	tm.t.Logf("MinerB RBP: %v, QaP: %v", p.MinerPower.QualityAdjPower.String(), p.MinerPower.RawBytePower.String())
```

Comment:
log line has `MinerB`, could either remove this entirely or replace it with `f0` actor address—which would be helpful because you can then correlate it with the order they were instantiated

Reply:
Fixed all these log lines.